src/registrar/templates/application_dotgov_domain.html

```diff
@@ -41,10 +39,7 @@ <h2 class="margin-top-3">Domain examples for your type of organization</h2>
       <h2>What .gov domain do you want?</h2>
```

Comment:
@Katherine-Osos I see you haven't changed this, but I'm noticing something funny in the way this h2 is displaying vs the h2 "Domain examples for your type of organization" 

they are two different sizes. The "What .gov domain you want?" h2 is 25.44px and the h2 "Domain examples for your type of organization" is at 24px. It seems like one may be getting optical sizing and the other one is a static 24px.  Might be a question for someone who knows the CSS though. @rachidatecs ? or @erinysong ?

Reply:
**What .gov domain do you want?**
<img width="594" alt="Screenshot 2024-01-17 at 10 07 44 AM" src="https://github.com/cisagov/manage.get.gov/assets/107440934/c851cf0f-61c3-416a-822b-af8c6720eced">


**Domain examples for your type of organization**
![Screenshot 2024-01-17 at 10 12 54 AM](https://github.com/cisagov/manage.get.gov/assets/107440934/403cef1b-d648-4332-a8a1-a490833272eb)